src/EventStore.Common/Configuration/ConfigurationRootExtensions.cs

```diff
@@ -1,11 +1,30 @@
 using System;
+using System.Linq;
 using Microsoft.Extensions.Configuration;
 
 #nullable enable
 namespace EventStore.Common.Configuration {
 	public static class ConfigurationRootExtensions {
-		public static string[] GetCommaSeparatedValueAsArray(this IConfigurationRoot configurationRoot, string key) =>
-			configurationRoot.GetValue<string?>(key)?.Split(',', StringSplitOptions.RemoveEmptyEntries) ??
-			Array.Empty<string>();
+		private static string[] INVALID_DELIMITERS = new []{ ";", "\t"};
+		public static string[] GetCommaSeparatedValueAsArray(this IConfigurationRoot configurationRoot, string key) {
+			string? value = configurationRoot.GetValue<string?>(key);
+			if (String.IsNullOrEmpty(value)) {
```

Comment:
```suggestion
			if (string.IsNullOrEmpty(value)) {
```

contentional to use the lower case alias